opm/simulators/utils/ParallelRestart.cpp

```diff
@@ -392,7 +392,7 @@ void pack(const std::vector<bool,A>& data, std::vector<char>& buffer, int& posit
           Dune::MPIHelper::MPICommunicator comm)
 {
     pack(data.size(), buffer, position, comm);
-    for (const auto& entry : data) {
+    for (const auto entry : data) { // Not a reference: vector<bool> range
```

Comment:
It should be possible to capture temporaries in const expressions. What was the warning?